src/backend/distributed/deparser/deparse_role_stmts.c

```diff
@@ -138,3 +156,42 @@ AppendAlterRoleStmt(StringInfo buf, AlterRoleStmt *stmt)
 		}
 	}
 }
+
+
+/*
+ * AppendAlterRoleStmt generates the string representation of the
+ * AlterRoleStmt and appends it to the buffer.
+ */
+static void
+AppendAlterRoleSetStmt(StringInfo buf, AlterRoleSetStmt *stmt)
+{
+	RoleSpec *role = stmt->role;
+	const char *roleSpecStr = NULL;
+
+	if (role == NULL)
+	{
+		/*
+		 * If all roles are be affected, role field is left blank in an
+		 * AlterRoleSetStmt.
+		 */
+		roleSpecStr = "ALL";
+	}
+	else
+	{
+		/*
+		 * If the role_specification used is CURRENT_USER or SESSION_USER,
+		 * it will be converted to thats roles role name.
+		 */
```

Comment:
Please add a line to the comment that  we pass `true` as a second argument causing the returned string to be quoted for direct use in the sql command below

Reply:
I updated the comment

https://github.com/citusdata/citus/blob/0e8103b101a207d9f52a2cb931d6dcee9e8c9828/src/backend/distributed/deparser/deparse_role_stmts.c#L181-L188